br/backup-and-restore-use-cases.md

```diff
@@ -91,6 +91,10 @@ For the detailed usage of the `br backup` command, refer to [BR command-line des
     UPDATE mysql.tidb SET VARIABLE_VALUE = '10m' WHERE VARIABLE_NAME = 'tikv_gc_life_time';
     ```
 
+> **Note:**
+>
+> Since v4.0.8, BR supports the self-adaptive GC. To avoid manually adjusting GC, register `backupTS` in `safePoint` in PD and make sure that `safePoint` does not move forward during the restore process.
```

Comment:
```suggestion
> Since v4.0.8, BR supports the self-adaptive GC. To avoid manually adjusting GC, register `backupTS` in `safePoint` in PD and make sure that `safePoint` does not move forward during the backup process.
```